src/System.Security.Cryptography.Xml/tests/Samples/SigningVerifyingX509Cert.cs

```diff
@@ -19,6 +19,8 @@ public class SigningVerifyingX509Cert
 <example>
 <test>some text node</test>
 </example>";
+        
+        private static bool SupportsDecryptValue => !PlatformDetection.IsFullFramework || PlatformDetection.IsNetfx462OrNewer();
```

Comment:
Not quite true... we still don't support DecryptValue.  SupportsNewRsaTypes, maybe?